bedrock/careers/templates/careers/includes/footer.html

```diff
@@ -3,53 +3,37 @@
  License, v. 2.0. If a copy of the MPL was not distributed with this
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
+{% block careers_bottom %}
+<section class="c-careers-bottom">
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed mzp-t-split-nospace',
+    media_class='mzp-l-split-h-center mzp-l-split-media-overflow',
+    media_include='careers/includes/big-picture-video.html',
+    image_url='img/careers/footer-remote.png',
+    include_highres_image=True,
+    media_after=True
+    ) %}
+    <h2>Ready for work that does you good?</h2>
+    <p>We’re hiring curious and kind people across the organization. Come help write Mozilla’s next chapter.</p>
+    <a href="{{ url('careers.listings') }}" class="mzp-c-button mzp-t-product">Browse open roles</a>
```

Comment:
Drive by nit: (was just browsing the demo site, nice job 👋): These blue `mzp-t-product` themed buttons are typically used for product links, like "Download Firefox". For consistency, could this alternatively be a regular black Mozilla themed link?

(Feel free to ignore, this is not a hill to die on, but i think consistency throughout the site can be important to consider).